src/Util.hs

```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Util where
+
+import           Prelude hiding (FilePath)
+import           Filesystem.Path
+import           Filesystem.Path.CurrentOS () -- https://github.com/fpco/haskell-filesystem/issues/11
+
+isBoring :: FilePath -> Bool
+isBoring p = ".git/" `elem` dirs || "dist/" `elem` dirs
```

Comment:
Personally, I did not have any issues with swap or backup files.  My assumption is that modifications to those files usually happen together with other, relevant, modifications.  Note that arbitrary modifications that happen within 100ms are treated as a single _trigger event_.  

I think in the worst case it may happen that we trigger too often.  Personally I prefer that over potentially missing modifications.  But I'm happy to address individual cases if we discover any in the future.

Thinking about it right now, one case where this may cause issues is if a test suite modifies files, `autospec` would go into look in that case.  Not sure how I want to address this.

Regarding inspecting cabal files, one of my design goals is not to depend at cabal at all.  Sandboxes are still supported by starting `autospec` with e.g.

    cabal exec autospec  test/Spec.hs